packages/fuels-test-helpers/src/node.rs

```diff
@@ -189,8 +227,32 @@ impl<'de> DeserializeAs<'de, BlockHeight> for HexNumber {
 
 pub fn get_node_config_json(
     coins: Vec<(UtxoId, Coin)>,
+    input_messages: Option<Vec<Message>>,
```

Comment:
Why `input_messages` instead of just `messages`? similar to how it's just `coins`?